BuildScript/BuildTasks/IPathHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.  You may obtain a copy of the
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+using System;
+
+namespace Remotion.BuildScript.BuildTasks
+{
+  public interface IPathHelper
+  {
+    string GetFileName (string fullPath);
```

Comment:
also, why do we need that one acutally? Can't we just use Path.xxx inline?

Reply:
I needed a mockable interface for testing. Or did I misunderstand your question?

Reply:
Yes. Why do we need to test this? Said another way, is there a benefit to mocking the file-name/directory-name parsing?

Reply:
I see your point that there is no benefit in testing Microsoft libraries, but the item's `Identity`, for example, is a string that uses the file name, and before I could write its logic I needed a test for it. I was thinking since the TestConfigurations promise to return the correct file information, it should also be tested that they do so, especially since the builds straight up die if the file information is incorrect.

Reply:
Hmm... it depends. If we later have a test/mock/whatever that requires a correctly parsed filename from a qualified path that was used as an input, then we would still be safe enough usign Path.GetFileName directly.